eng/mgmt/mgmtmetadata/automation_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: v2
```

Comment:
this file is automation_resource-manager.txt indicating it was not from running the generate.ps1. That file needs to be corrected. This file should have been **automanage**_resource-manager.txt

Reply:
Reran generate and updated, thank you.